packages/pr-antora-content-guidelines-checker/action.yml

```diff
@@ -0,0 +1,23 @@
+name: 'PR Antora content guidelines checker' pr-antora-content-guideline-checker
+description: 'Checks files updated on PR are following the contribution guideline for an Antora documentation content.'
+inputs:
+  github-token:
+    description: 'The token used to create Pull Request comment when the contribution does not follow requirements'
+    required: false
+    default: '${{ github.token }}'
```

Comment:
**thought:** the quote may be the cause of the current problem: https://github.com/bonitasoft/actions/pull/115#issuecomment-1770261585
Here is what we use elsewhere https://github.com/bonitasoft/actions/pull/115#discussion_r1358368807